css/templates/themes/2/page-vars.css

```diff
@@ -50,7 +50,7 @@
   --color-navbar-primary: var(--color-black);
   --color-navbar-font-primary: var(--color-white);
 
-  --color-input-primary: var(--color-font-accent);
+  --color-input-primary: var(--color-secondary);
```

Comment:
yes , it should be a color that fits the theme (i don't know what it is until i download your patch and check - if you can upload an img i can verify)